calibration_with_blender/calibration_benchmark.cpp

```diff
@@ -0,0 +1,684 @@
+#include "opencv2/core.hpp"
```

Comment:
I do not think that calibration.cpp copy-paste from main repo is required. Could we just reuse it?